src/providers/seventv/SeventvBadges.cpp

```diff
@@ -0,0 +1,72 @@
+#include "providers/seventv/SeventvBadges.hpp"
+
+#include "common/NetworkRequest.hpp"
+#include "common/Outcome.hpp"
+#include "messages/Emote.hpp"
+
+#include <QUrl>
+#include <QUrlQuery>
+
+#include <map>
+
+namespace chatterino {
+
+void SeventvBadges::initialize(Settings & /*settings*/, Paths & /*paths*/)
+{
+    this->loadSeventvBadges();
+}
+
+boost::optional<EmotePtr> SeventvBadges::getBadge(const UserId &id)
+{
+    auto it = badgeMap_.find(id.string);
+    if (it != badgeMap_.end())
+    {
+        return emotes_[it->second];
+    }
+    return boost::none;
+}
+
+void SeventvBadges::loadSeventvBadges()
+{
+    // Cosmetics will work differently in v3, until this is ready
+    // we'll use this endpoint.
+    static QUrl url("https://api.7tv.app/v2/badges");
```

Comment:
Can this be changed to `/v2/cosmetics`? Badges is an alias that has been deprecated for a while

Reply:
Also, use `7tv.io` as base instead of `api.7tv.app`

Reply:
sure